src/modules/xcm/index.ts

```diff
@@ -350,6 +350,15 @@ export let xcmChainObj: XcmChainObj = {
     subscan: 'https://unique.subscan.io',
     isAstarNativeToken: false,
   },
+  [Chain.PENDULUM]: {
+    name: Chain.PENDULUM,
+    relayChain: Chain.POLKADOT,
+    img: require('/src/assets/img/token/pen.png'),
```

Comment:
How about we add the image as an SVG instead so that it's easily scalable.

Reply:
I didn't see a PEN.svg in our drive...?
https://drive.google.com/drive/u/0/folders/1sZcQzoDxT_EUd2wtEJN8ECNzPakfAMs6 only PNG. 
I'll ask Anna in the ticket to provide us one.